<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customerLogin</class>
 <widget class="QDialog" name="customerLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Product Order Form&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backLogin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back To Login Screen</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>191</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Product Name</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Secure Email Gateway</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>341</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Description</string>
   </property>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>321</width>
      <height>81</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Roboto','arial','sans-serif'; font-size:12pt; color:#222222;&quot;&gt;A &lt;/span&gt;&lt;span style=&quot; font-family:'Roboto','arial','sans-serif'; font-size:12pt; font-weight:600; color:#222222;&quot;&gt;Secure Email Gateway&lt;/span&gt;&lt;span style=&quot; font-family:'Roboto','arial','sans-serif'; font-size:12pt; color:#222222;&quot;&gt; (SEG) is a device or software used to monitor &lt;/span&gt;&lt;span style=&quot; font-family:'Roboto','arial','sans-serif'; font-size:12pt; font-weight:600; color:#222222;&quot;&gt;emails&lt;/span&gt;&lt;span style=&quot; font-family:'Roboto','arial','sans-serif'; font-size:12pt; color:#222222;&quot;&gt; that are being sent and received. An SEG is designed to prevent unwanted &lt;/span&gt;&lt;span style=&quot; font-family:'Roboto','arial','sans-serif'; font-size:12pt; font-weight:600; color:#222222;&quot;&gt;email&lt;/span&gt;&lt;span style=&quot; font-family:'Roboto','arial','sans-serif'; font-size:12pt; color:#222222;&quot;&gt; and deliver good &lt;/span&gt;&lt;span style=&quot; font-family:'Roboto','arial','sans-serif'; font-size:12pt; font-weight:600; color:#222222;&quot;&gt;email&lt;/span&gt;&lt;span style=&quot; font-family:'Roboto','arial','sans-serif'; font-size:12pt; color:#222222;&quot;&gt;. Messages that are unwanted include spam, phishing attacks, malware or fraudulent content.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:120%; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Roboto','arial','sans-serif'; font-size:12pt; color:#222222;&quot;&gt;&lt;br /&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>80</y>
     <width>120</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Order</string>
   </property>
   <widget class="QRadioButton" name="secureRadio">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Encryption</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Server Security</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>341</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTextBrowser" name="textBrowser_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>81</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Roboto','arial','sans-serif'; font-size:12pt; color:#222222; background-color:#ffffff;&quot;&gt;In cryptography, encryption is the process of encoding a message or information in such a way that only authorized parties can access it and those who are not authorized cannot. Encryption does not itself prevent interference, but denies the intelligible content to a would-be interceptor.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>220</y>
     <width>120</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="dataRadio">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>341</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTextBrowser" name="textBrowser_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>81</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Roboto','arial','sans-serif'; font-size:13px; color:#222222; background-color:#ffffff;&quot;&gt;Web &lt;/span&gt;&lt;span style=&quot; font-family:'Roboto','arial','sans-serif'; font-size:13px; font-weight:600; color:#222222; background-color:#ffffff;&quot;&gt;server security&lt;/span&gt;&lt;span style=&quot; font-family:'Roboto','arial','sans-serif'; font-size:13px; color:#222222; background-color:#ffffff;&quot;&gt; is the protection of information assets that can be accessed from a Web &lt;/span&gt;&lt;span style=&quot; font-family:'Roboto','arial','sans-serif'; font-size:13px; font-weight:600; color:#222222; background-color:#ffffff;&quot;&gt;server&lt;/span&gt;&lt;span style=&quot; font-family:'Roboto','arial','sans-serif'; font-size:13px; color:#222222; background-color:#ffffff;&quot;&gt;. Web &lt;/span&gt;&lt;span style=&quot; font-family:'Roboto','arial','sans-serif'; font-size:13px; font-weight:600; color:#222222; background-color:#ffffff;&quot;&gt;server security&lt;/span&gt;&lt;span style=&quot; font-family:'Roboto','arial','sans-serif'; font-size:13px; color:#222222; background-color:#ffffff;&quot;&gt; is important for any organization that has a physical or virtual Web &lt;/span&gt;&lt;span style=&quot; font-family:'Roboto','arial','sans-serif'; font-size:13px; font-weight:600; color:#222222; background-color:#ffffff;&quot;&gt;server&lt;/span&gt;&lt;span style=&quot; font-family:'Roboto','arial','sans-serif'; font-size:13px; color:#222222; background-color:#ffffff;&quot;&gt; connected to the Internet.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Roboto','arial','sans-serif'; font-size:12pt; color:#222222;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_9">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>340</y>
     <width>120</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="serverRadio">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_10">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>80</y>
     <width>120</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Price</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>$4000 Annually</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_11">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>220</y>
     <width>120</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>$5000 Annually</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_12">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>340</y>
     <width>120</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>$10000 Annually</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="ConfirmPurchase">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>500</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm Purchase</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="totalLCD">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>510</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>510</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;$&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Status4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>[+]Status</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>backLogin</sender>
   <signal>clicked()</signal>
   <receiver>customerLogin</receiver>
   <slot>buttonBackLogin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonBackLogin()</slot>
 </slots>
</ui>
